docs/api-reference/organization.md

```diff
@@ -25,7 +25,7 @@ Like `Organization#app()`, but as a subscription.
 | `appFilter.address` | `String`                      | Same as `appFilter`, but makes the selection by `address` explicit.                                                                                                                                                                                           |
```

Comment:
I wonder what you think of using the types (`Address` and `SubscriptionHandler`) on the documentation as well. And maybe we should have a new documentation page with all the types we currently have?

Reply:
Yes definitely, I think we could have a page for the types somewhere. :+1: